packages/EasyCore/tests/Doctrine/EntityManagers/EntityManagerDecoratorTest.php

```diff
@@ -0,0 +1,386 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Doctrine\ORM\Decorators;
```

Comment:
Pls move this file to the corresponding folder.

Reply:
Done